tutorials/notebooks/FITS-cubes/FITS-cubes.ipynb

```diff
@@ -360,18 +362,20 @@
     "fig = plt.figure(figsize=(18, 12))\n",
     "\n",
     "# Initiate a FITSFigure to set up axes\n",
-    "F = aplpy.FITSFigure(moment_1.hdu, figure=fig)\n",
+    "#F = aplpy.FITSFigure(moment_1.hdu, figure=fig)  # Please uncomment to initiate\n",
```

Comment:
I think you need to uncomment the line above. The error at https://circleci.com/gh/astropy/astropy-tutorials/472?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link happens on line 371 below because `F` was never created because this line was commented out.

Reply:
Yeah, for some reason Circle CI doesn't like some of the `aplpy` syntax... I had to fix the error in a dumb way by commenting out all invocations of `F`... At least the test is passing now. Do you think there is something we can do with Circle CI to prevent such occurrences, i.e. the hyper-sensitivity to some non-standard package syntax?